### PR TITLE
Change recommended visibility level for the command handler methods

### PR DESCRIPTION
**Pull request by IlliaShepilov**
Summary of changes:

- Changed recommended visibility level for the command handler methods
- Fixed documentation in the Assign annotation
- Changed recommended visibility level in the test classes

### CODE REVIEW

File: server/src/main/java/org/spine3/server/reflect/CommandHandlerMethod.java [hunk 1]
```diff
@@ -130,8 +130,12 @@ public CommandHandlerMethod create(Method method) {
 
         @Override
         public void checkAccessModifier(Method method) {
-            if (!Modifier.isPublic(method.getModifiers())) {
-                warnOnWrongModifier("Command handler method {} should be declared 'public'.", method);
+            final int modifiers = method.getModifiers();
+            final boolean nonDefaultModifier =
+                    Modifier.isPublic(modifiers) || Modifier.isProtected(modifiers) || Modifier.isPrivate(modifiers);
```
Comment (armiol) [inline server/src/main/java/org/spine3/server/reflect/CommandHandlerMethod.java]: `static import`?

File: server/src/main/java/org/spine3/server/reflect/CommandHandlerMethod.java [hunk 2]
```diff
@@ -130,8 +130,12 @@ public CommandHandlerMethod create(Method method) {
...
+            if (nonDefaultModifier) {
+                warnOnWrongModifier("Command handler method {} should be declared with default access modifier.",
```
Comment (armiol) [inline server/src/main/java/org/spine3/server/reflect/CommandHandlerMethod.java]: `the default`.

**Comment by IlliaShepilov** [comment]
@armiol please take a look at the updates.

File: server/src/main/java/org/spine3/server/command/Assign.java [hunk 3]
```diff
@@ -35,7 +35,7 @@
  * <p>A command handler method:
  * <ul>
  *     <li>is annotated with {@link Assign};
- *     <li>is {@code public};
+ *     <li>has default access modifier;
```
Comment (armiol) [inline server/src/main/java/org/spine3/server/command/Assign.java]: `has a default access modifier`.

File: build.gradle [hunk 4]
```diff
@@ -33,7 +33,7 @@ buildscript {
         guavaVersion = '20.0'
         protobufGradlePluginVerison = '0.8.0'
 
-        spineVersion = '0.7.16-SNAPSHOT'
+        spineVersion = '0.7.17-SNAPSHOT'
```
Comment (armiol) [inline build.gradle]: Please browse the active branches. I think 0.7.17-SNAPSHOT may already be used. Probably, you'll have to increase the version even more.

**Pull request merged**